packages/js/plugins/uri-resolvers/ipfs-resolver/src/schema.graphql

```diff
@@ -1,5 +1,26 @@
 #import { Module, MaybeUriOrManifest } into UriResolver from "ens/uri-resolver.core.polywrap.eth"
 #import { Module } into Ipfs from "ens/ipfs.polywrap.eth"
 
+type Timeouts {
+  getFile: UInt32,
+  checkIfExists: UInt32,
+  tryResolveUri: UInt32,
+}
+
+type Env {
+    """
+    Determines whether the plugin should try to resolve a file (check for its existence) or immediately get the file
+    """
+    skipCheckIfExistsBeforeGetFile: Boolean,
```

Comment:
**Brainstorming other names**
skipCheckIfExists
skipFileCheck
skipCheckBeforeGetFile

I think the name you have already is long but clear and informative. If you decide to change it, my favorite alternative is `skipCheckIfExists` because it matches the timeout name.

Reply:
I think `skipCheckIfExists` sounds best too! Will go with that one.